packages/utils/test/testutils.ts

```diff
@@ -11,3 +11,7 @@ export const testOnlyIfNodeVersionAtLeast = (minVersion: number): jest.It => {
 
   return it;
 };
+
+export function parseEnvelope(env: string): Array<Record<any, any>> {
+  return env.split('\n').map(e => JSON.parse(e));
+}
```

Comment:
FWIW, we have such a function already if you want to use it: https://github.com/getsentry/sentry-javascript/blob/bb6f86502e1ea551fa503136a954844f77cca322/packages/core/test/lib/request.test.ts#L22-L30. Up to you.

Reply:
I'll leave it like this because this works with multiple envelope items - doesn't just assume a single one. Useful for when we expand the tests in the future.